### DISCUSSION TITLE
Unable to authenticate my reddit user?

### DISCUSSION THREAD

**Discussion by FowadIjaz** [Serene-Arc/bulk-downloader-for-reddit#809] [Q&A]
I've been able to use BDFR before, but had to do a new install of python which meant reinstalling BDFR. This means I'm on python 3.10 and the latest version of BDFR, 2.6 I think. Problem is that when I use the flag --authenticate, it returns the following error:

```[2023-03-04 17:42:16,510 - bdfr.connector - ERROR] - User username failed to be retrieved due to a PRAW exception: received 400 HTTP response```

I've completely forgotten what the process to authenticate is.. I've gone back to the older version to see if I can piece it together, but get the same error message.  Can anyone help?

**Comment by Serene-Arc** [comment]
Try running the `--authenticate` option without any other arguments, just that on the command line. That should let you go through the process in a simpler manner.

This error seems to be an issue with the command you're using. The BDFR is looking for a user named 'username', which doesn't exist, so it's erroring out.

**Reply by FowadIjaz** [reply]
Sorry, forgot to mention that I'm not using a generic user named username, I just switched out my actual username to that in the command. 

For running `--authenticate` by iteself, I'm not sure how that works, would it be like this?

```
python -m bdfr clone ./Test --user username --authenticate
```

**Comment by FowadIjaz** [comment]
So... the issue was that I was already logged in as another user. Or anyways, the program was relying on authentication data other than the default data. Using the bash script in this issue, I was able to "log out" and then the authentication process went through smoothly : https://github.com/aliparlakci/bulk-downloader-for-reddit/issues/604
